qutebrowser/browser/commands.py

```diff
@@ -1226,7 +1253,13 @@ def home(self):
         """Open main startpage in current tab."""
         self.openurl(config.val.url.start_pages[0])
 
-    def _run_userscript(self, cmd, *args, verbose=False):
+    def _selection_callback(self, s):
```

Comment:
This seems like a quite generic name - why not something like `_run_userscript_wrapper` or so?